.github/workflows/release.yml

```
@@ -20,7 +20,7 @@ env:
 jobs:
   fetch-grammars:
     name: Fetch Grammars
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

this one can still be run on latest since it is only cloning the grammar repositories

```suggestion
    runs-on: ubuntu-latest
```